packages/terra-demographics-banner/lib/DemographicsBanner.scss

```diff
@@ -0,0 +1,49 @@
+/* stylelint-disable  */
+
+@import '~terra-mixins';
+@import './variables';
+@import './mixins';
+
+.terra-DemographicsBanner {
+  background: $demographics-background;
+  color: $demographics-foreground;
+  display: flex;
+  margin-bottom: 0.714em;
+  padding: 0.714em;
+}
+
+.terra-DemographicsBanner--person-name {
```

Comment:
nit: I think these components are decendants for the demographics banner: https://github.cerner.com/standards/rfc/blob/master/published/0004-css-naming-convention.md#descendent
```
.terra-DemographicsBanner--person-name
```
to
``` 
terra-DemographicsBanner-person-name
```